src/components/filters/FilterList/index.jsx

```diff
@@ -85,47 +85,64 @@ class FilterList extends React.Component {
     // Takes in parent component's filterStatus or self state's filterStatus
     const filterStatus = this.props.filterStatus
       ? this.props.filterStatus : this.state.filterStatus;
-
+    const filterSectionToShow = [];
+    this.props.sections.forEach((section, index) => {
+      if (this.props.hideEmptyFilterSection && section.options.length === 0) {
+        return;
+      }
+      filterSectionToShow.push(
+        <FilterSection
+          key={index}
+          ref={this.sectionRefs[index]}
+          title={section.title}
+          tooltip={section.tooltip}
+          options={section.options}
+          isSearchFilter={section.isSearchFilter}
+          isArrayField={section.isArrayField}
+          onSearchFilterLoadOptions={section.onSearchFilterLoadOptions}
+          expanded={this.props.expandedStatus[index]}
+          onToggle={newExpanded => this.handleSectionToggle(index, newExpanded)}
+          onClear={() => this.handleSectionClear(index)}
+          filterStatus={filterStatus[index]}
+          onSelect={
+            singleFilterLabel => this.handleSelectSingleFilter(
+              index,
+              singleFilterLabel,
+            )
+          }
+          onCombineOptionToggle={
+            (combineModeFieldName, combineModeValue) => this.handleSelectCombineOptionToggle(
+              index,
+              combineModeFieldName,
+              combineModeValue,
+            )
+          }
+          onAfterDrag={
+            (...args) => this.handleDragRangeFilter(index, ...args)
+          }
+          hideZero={this.props.hideZero}
+          tierAccessLimit={this.props.tierAccessLimit}
+          lockedTooltipMessage={this.props.lockedTooltipMessage}
+          disabledTooltipMessage={this.props.disabledTooltipMessage}
+        />,
+      );
+    });
+    // if no filters in tab add message
+    if (filterSectionToShow.length === 0) {
+      filterSectionToShow.push(
+        <div key='g3-filter-list-empty' className='g3-filter-list-empty g3-filter-section'>
+          <h4>
+            Selected Data Does NOT Include
+            <br />
+            These Filters
+          </h4>
+          <span>Disable Filters in other tabs to view</span>
```

Comment:
What if, say, data explorer is in `private` mode, and a user with no access to anything logged in? Would he/she just see a bunch of empty filter lists with this message? If so, lets remove this sentence, because it will be confusing to users

Reply:
```suggestion
```